tests/acceptance/interactions-test.js

```diff
@@ -156,3 +157,17 @@ test('#fillInByName fills input by name', (assert) => {
     assert.equal(val, targetValue, 'expect the input to have the target value');
   });
 });
+
+test('#chooseFromDropdown selects a dropdown option by option', (assert) => {
```

Comment:
Perhaps change to say "...dropdown option by option label" since the helper expects the actual option text